editor/post-title/index.js

```diff
@@ -88,12 +121,15 @@ class PostTitle extends Component {
 
 	render() {
 		const { title } = this.props;
-		const { isSelected } = this.state;
-		const className = classnames( 'editor-post-title', { 'is-selected': isSelected } );
```

Comment:
Maybe isSelected should be something like isTyping?

Reply:
The naming had confused me as well when I first encountered it. I think `isTyping` would be an improvement, yes.